kit/src/components/embeds/file_embed/mod.rs

```diff
@@ -244,3 +300,39 @@ pub fn FileEmbed<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
         }
     ))
 }
+
+fn get_file_thumbnail_if_is_image(filepath: PathBuf, filename: String) -> String {
```

Comment:
This shouldnt be needed since the type and thumbnail is determined from within constellation unless there is some other use for it (see https://github.com/Satellite-im/Uplink/pull/828)